### PR TITLE
container: initialize results list

### PR DESCRIPTION
**Pull request by giuseppe**
it solves:

$ podman ps --format=json
null

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: cmd/podman/shared/container.go [hunk 1]
```diff
@@ -449,10 +449,8 @@ func GetPsContainerOutput(r *libpod.Runtime, opts PsOptions, filters []string, m
 // PBatch performs batch operations on a container in parallel. It spawns the
 // number of workers relative to the number of parallel operations desired.
 func PBatch(containers []*libpod.Container, workers int, opts PsOptions) []PsContainerOutput {
-	var (
-		wg        sync.WaitGroup
-		psResults []PsContainerOutput
-	)
+	var wg sync.WaitGroup
+	psResults := []PsContainerOutput{}
```
Comment (TomSweeneyRedHat) [inline cmd/podman/shared/container.go]: interesting fix, nice catch @giuseppe

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
/lgtm
/hold
hold cancel when tests pass.

**Pull request merged**